### PR TITLE
Roughly scan the formula related parts and did some modification

### PR DESCRIPTION
**Pull request by dongh11**

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-zh/PR-613/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-613/

File: chapter_convolutional-modern/alexnet.md [hunk 1]
```diff
@@ -3,18 +3,19 @@
 
 尽管在LeNet的到来后，卷积神经网络在计算机视觉和机器学习领域中很有名。但卷积神经网络并没有主导这个领域。这是因为虽然 LeNet 在小数据集上取得了很好的效果，但是在更大、更真实的数据集上训练卷积神经网络的性能和可行性还有待研究。事实上，在上世纪90年代初到2012年之间的大部分时间里，神经网络往往被其他机器学习方法所超越，如支持向量机（support vector machines）。
 
-在计算机视觉中，直接将神经网络与其他机器学习方法进行比较也许不公平。卷积神经网络的输入由原始像素值或经过轻微预处理（例如居中、缩放）的像素值组成。但使用传统机器学习时，从业者永远不会将原始像素作为输入。传统机器学习的计算机视觉管道是由经过人工精心设计的特征管道组成。具体的说，传统机器学习方法有以下几个步骤：
+在计算机视觉中，直接将神经网络与其他机器学习方法进行比较也许不公平。这是因为，卷积神经网络的输入是由原始像素值或是经过简单预处理（例如居中、缩放）的像素值组成的。但在使用传统机器学习方法时，从业者永远不会将原始像素作为输入。传统机器学习方法的计算机视觉管道是由经过人的手工精心设计的特征管道组成的。对于这些传统方法，大部分的进展都来自于对特征有了更聪明的想法，并且学习算法往往归于事后的解释。
```
Comment (goldmermaid) [inline chapter_convolutional-modern/alexnet.md:6]: "传统机器学习方法的计算机视觉管道是由经过人的手工精心设计的特征管道组成的。" 请分成短句

File: chapter_deep-learning-computation/parameters.md [hunk 2]
```diff
@@ -246,20 +246,20 @@ rgnet.layers[0].layers[1].layers[1].weights[1]
 我们知道了如何访问参数，现在让我们看看如何正确地初始化参数。我们在 :numref:`sec_numerical_stability` 中讨论了良好初始化的必要性。深度学习框架提供默认随机初始化。然而，我们经常希望根据其他规则初始化权重。深度学习框架提供了最常用的规则，也允许创建自定义初始化方法。
 
 :begin_tab:`mxnet`
-默认情况下，MXNet通过初始化权重参数的方法是从均匀分布$U(-0.07, 0.07)$随机采样权重，并将偏差参数设置为0。MXNet的`init`模块提供了多种预置初始化方法。
+默认情况下，MXNet通过初始化权重参数的方法是从均匀分布$U(-0.07, 0.07)$中随机采样权重，并将偏置参数设置为0。MXNet的`init`模块提供了多种预置初始化方法。
```
Comment (goldmermaid) [inline chapter_deep-learning-computation/parameters.md:249]: “偏差参数”和“偏置参数”可以全文align一下嘛？哪个比较常用呢？

**Comment by mli** [comment]
Job d2l-zh/PR-613/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-613/

**Comment by mli** [comment]
Job d2l-zh/PR-613/3 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-613/

**Comment by mli** [comment]
Job d2l-zh/PR-613/4 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-613/

**Comment by mli** [comment]
Job d2l-zh/PR-613/5 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-613/

**Pull request merged**